cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -448,11 +448,7 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) caerrors.AutoscalerErr
 		return nil
 	}
 
-	danglingNodes, err := a.deleteCreatedNodesWithErrors()
-	if err != nil {
-		klog.Warningf("Failed to remove nodes that were created with errors, skipping iteration: %v", err)
-		return nil
-	}
+	danglingNodes := a.deleteCreatedNodesWithErrors()
```

Comment:
nit: maybe inline this within the condition to make it clear we only use this var here and don't need it later? sth like:

```
if deletedSth := a.deleteCreatedNodesWithErrors(); deletedSth {
   ...
}
```

(The var is of course superfluous, but makes it clearer what's happening here.)

Reply:
Done